test/Conversion/TorchOnnxToTorch/ops/lstm.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: torch-mlir-opt <%s --split-input-file -convert-torch-onnx-to-torch | FileCheck %s
+
+
+
+// CHECK-LABEL:   func.func @test_lstm_basic(
+// CHECK-SAME:                               %[[X:.*]]: !torch.vtensor<[15,2,4],f32>,
+// CHECK-SAME:                               %[[W:.*]]: !torch.vtensor<[1,12,4],f32>,
+// CHECK-SAME:                               %[[R:.*]]: !torch.vtensor<[1,12,3],f32>,
+// CHECK-SAME:                               %[[B:.*]]: !torch.vtensor<[1,24],f32>) -> (!torch.vtensor<[15,1,2,3],f32>, !torch.vtensor<[1,2,3],f32>, !torch.vtensor<[1,2,3],f32>) attributes {torch.onnx_meta.ir_version = 9
```

Comment:
Strip unneeded content from checks. E.g. version information and return types (as they aren't changed)